fuel-vm/src/interpreter/memory/allocation_tests.rs

```diff
@@ -150,141 +150,6 @@ fn test_stack_pointer_overflow(
     Ok(())
 }
 
-#[test_case(20, 20 => Ok(()); "Can load a byte")]
```

Comment:
Why were these tests removed?

Reply:
They were testing the internal implementation details, instead of the actual functionality. There is [ongoing effort](https://github.com/FuelLabs/fuel-vm/issues/746) to switch away from these anyway, so this was the natural moment to do that. The instructions are already tested in `fuel-vm/src/tests/memory.rs`.